### DISCUSSION TITLE
Should PyFluent clean up Fluent .trn files?

### DISCUSSION THREAD

**Discussion by raph-luc** [ansys/pyfluent#2000] [Ideas]
Minor issue/idea after talking with Adam. In addition to cleaning up the Fluent process itself, we could, maybe by default, also clean up the Fluent `.trn` files as they are often not wanted

**Comment by mkundu1** [comment]
We can set the env var `FLUENT_NO_AUTOMATIC_TRANSCRIPT=1` within PyFluent to disable automatic transcript generation.

**Reply by seanpearsonuk** [reply]
@mkundu1 Shall we expose the option to create a transcript file as a launch argument, and implement it through the environment variable?

**Reply by mkundu1** [reply]
@seanpearsonuk One thing is that the automatic transcript in Fluent is not relevant for PyFluent users, so that doesn't have to be exposed in PyFluent API. We can have a module-level global for it which will be less prominent.

Similar treatment was done for `pyfluent.SHOW_MESH_AFTER_CASE_READ`.

**Comment by raph-luc** [comment]
True, I think that is a good suggestion, adding the option to disable auto transcript start. This is probably better than letting Fluent write the .trn file, only to delete it on exit. Assuming this doesn't interfere with the transcript streaming we currently use
